telebot/__init__.py

```diff
@@ -8110,11 +8127,17 @@ def register_business_message_handler(self,
         :param callback: function to be called
         :type callback: :obj:`function`
 
+        :param commands: list of commands
+        :type commands: :obj:`list` of :obj:`str`
+
+        :param regexp: Regular expression
+        :type regexp: :obj:`str`
+
         :param func: Function executed as a filter
         :type func: :obj:`function`
 
-        :param pass_bot: True if you need to pass TeleBot instance to handler(useful for separating handlers into different files)
-        :type pass_bot: :obj:`bool`
+        :param content_types: Supported message content types. Must be a list. Defaults to ['text'].
```

Comment:
does it?...
we are not setting content_types

Reply:
![image](https://github.com/eternnoir/pyTelegramBotAPI/assets/5613421/5fdfebec-f726-4317-949a-d6f52bcffca4)

I just align description with parameters.

Reply:
Ah ok, seems like we need another full PR with all content_types description fixes + default values. you can merge